src/boot/efi/meson.build

```diff
@@ -257,6 +257,7 @@ efi_ldflags = [
         '-Wl,--warn-common',
         '-Wl,-Bsymbolic',
         '-z', 'nocombreloc',
+        '-Wl,-z,noexecstack',
```

Comment:
Maybe `'-z', 'noexecstack'`, while being the exact equivalent of `'-Wl,-z,noexecstack'`, would look nicer.